packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +57,309 @@ export default {
- * 2. Set an empty string to clear the custom validity and make the input valid.
- * 3. To show the validation message, call the `reportValidity` method. Originally this would show a native validation bubble, but we show the error messages inline.
+ * The component gets `valid` state when the input is valid.<br />Use the `style-on-valid` attribute to automatically indicate and show a valid state.
  */
 
-export const setCustomValidity = {
-  parameters: {
-    chromatic: { disableSnapshot: true }
-  },
-  render: () => {
-    return html`
-      <!-- block submit and show alert instead -->
-      <form id="validationForm" class="flex flex-col gap-2">
-        <sd-select id="custom-input" style-on-valid>
-          <sd-option value="option-1">Option 1</sd-option>
-          <sd-option value="option-2">Option 2</sd-option>
-          <sd-option value="option-3">Option 3</sd-option>
-        </sd-select>
-        <div>
-          <sd-button type="submit">Submit</sd-button>
-          <sd-button id="error-button" variant="secondary">Set custom error</sd-button>
-          <sd-button id="success-button" variant="secondary">Set success</sd-button>
-          <sd-button type="reset" variant="secondary">Reset</sd-button>
-        </div>
-      </form>
-      <script type="module">
-        // Wait for custom elements to be defined
-        await Promise.all([
-          customElements.whenDefined('sd-select'),
-          customElements.whenDefined('sd-button'),
-          customElements.whenDefined('sd-option')
-        ]).then(() => {
-          const form = document.getElementById('validationForm');
-          const input = document.getElementById('custom-input');
-          const setErrorButton = document.getElementById('error-button');
-          const setSuccessButton = document.getElementById('success-button');
-
-          // Initial error
-          const errorMessage = \`This is an initial custom error (\${new Date().toLocaleTimeString()})\`;
-          input.setCustomValidity(errorMessage);
-          input.reportValidity();
-
-          // Show error message
-          setErrorButton.addEventListener('click', () => {
-            const errorMessage = \`This is a new custom error (\${new Date().toLocaleTimeString()})\`;
-            input.setCustomValidity(errorMessage);
-            input.reportValidity();
-          });
-
-          // Show success message
-          setSuccessButton.addEventListener('click', () => {
-            input.setCustomValidity(''); // Clear custom validity
-            input.reportValidity();
-          });
-
-          form.addEventListener('submit', event => {
-            event.preventDefault();
-            alert('All fields are valid!');
-          });
-        });
-      </script>
-    `;
-  }
+export const Valid = {
+  render: () => html`
+    <div class="w-[500px] h-[300px]">
```

Comment:
Valid should look like this:

![image](https://github.com/user-attachments/assets/f556e2bd-fc47-4b69-91e3-1d33fa7302ba)

Hide clearable and keep Option 1 in color text.black

Reply:
Currently the value color is hardcoded. Will be addressed by [#1172](https://github.com/solid-design-system/solid/issues/1172)